crates/tools/python/README.md

```diff
@@ -0,0 +1,16 @@
+# iPython Notebook
+
+## Requirements
+
+- [Miniconda](https://docs.anaconda.com/free/miniconda/index.html)
+
+## Running
+
+From this directory:
+
+```bash
+conda env create -f env.yml
```

Comment:
If you use `conda`, you might be interested in https://pixi.sh/latest/

Reply:
Thanks, I just use `conda` for reproducible ipython notebooks.